src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$main-padding: 16px;
+
+@mixin centered-flexbox {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+@import "./reset.scss";
+
 body {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.card {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
Move to the * selector and remove from all other places.

Reply:
if I remove box-sizing border box, the card starts being compressed, so I keep it in .card selector